native/src/format.rs

```diff
@@ -463,6 +464,9 @@ pub fn use_parens_for_method_call(
     }
 
     if name == "return" || name == "raise" || name == "yield" || name == "break" {
+        if ps.current_formatting_context() == FormattingContext::Binary {
```

Comment:
What do you think about adding a method for this to avoid coupling to the structure?

```suggestion
        if ps.current_formatting_context().is_binary() {
```

And the method would look like

```
fn is_binary(&self) -> bool {
    match self {
        FormattingContext::Binary => true,
        _ => false,
    }
}
```